SocialFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Extended CSS rules
 !
+zaytung.com#?#div[width="630px"][align="right"] > a[href]:contains(Paylaş)
```

Comment:
Try without using ext-css `zaytung.com##div[width][align="right"] > a[style^="color:#60B7B5;"]`. According to their code, it should work fine.

Reply:
In this case, I prefer ext-css.

Reply:
Why so? The basic rule works fine. It's not really necessary to use ext-css here.